rest_api/.jhipster/Subscription.json

```diff
@@ -27,10 +27,15 @@
       "fieldId": 1,
       "fieldName": "subscriptionId",
       "fieldType": "Long"
+    },
+    {
+      "fieldId": 2,
+      "fieldName": "topics",
```

Comment:
I will change the field name to topicFilterString. I noticed that MYSQL table is not getting updated with the new field. I will make necessary changes and commit again.